scene/gui/text_edit.h

```diff
@@ -809,6 +810,9 @@ class TextEdit : public Control {
 	void set_caret_blink_interval(const float p_interval);
 	float get_caret_blink_interval() const;
 
+	void set_draw_caret_when_readonly_enabled(bool p_value);
+	bool is_drawing_caret_when_readonly_enabled();
```

Comment:
Missing the `const` here.

Reply:
Hmm, I'm getting an error saying that my setter is invalid. Can you help me? Sorry, I'm trying to find but not being able to.